### PR TITLE
Add initial Django configuration

### PR DESCRIPTION
**Pull request by schenkh**
Added initial Django configuration and modified pylint configuration in pre-commit to run locally
Relates to #21

### CODE REVIEW

**Comment by webknjaz** [comment]
Add a link back to the issue into the PR description.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -46,4 +46,20 @@ repos:
 -   repo: https://github.com/pre-commit/mirrors-pylint
     rev: v2.3.1
     hooks:
-    -   id: pylint
+    - id: pylint
```
Comment (webknjaz) [inline .pre-commit-config.yaml:49]: This change is unrelated to what's stated in the PR title. Please make atomic changes as it's important for the project maintainability.

Comment (webknjaz) [reply .pre-commit-config.yaml:49]: Submit it separately.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -46,4 +46,20 @@ repos:
...
+      args:
+      - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+      - --output-format=colorized
+      - |-
+        --init-hook=
+        import os, platform, sys;
+        sys.path.insert(
+            0,
+            "".join([os.getenv("HOME"), "/virtualenv/python", platform.python_version(),
```
Comment (webknjaz) [inline .pre-commit-config.yaml:58]: Please don't use this hack and use a local mode as asked in the original task https://github.com/kpi-web-guild/django-girls-blog-schenkh/issues/7

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 2 above)
Comment (schenkh) [reply .pre-commit-config.yaml:58]: I was pretty sure I did that, let me check..

File: .pre-commit-config.yaml (hunk 1 above)
Comment (schenkh) [reply .pre-commit-config.yaml:49]: So new PR for this specific change, right?

Comment (webknjaz) [reply .pre-commit-config.yaml:49]: yep
